src/traits.rs

```diff
@@ -733,6 +733,17 @@ pub trait SystemExt: Sized + Debug + Default {
     /// ```
     fn get_processors(&self) -> &[Processor];
 
+    /// Returns the physical numer of cores on the processor.
+    /// In the case of multiple CPUs it will combine the physical core count of all the CPUs
+    ///
+    /// ```no_run
+    /// use sysinfo::{ProcessorExt, System, SystemExt};
+    ///
+    /// let s = System::new();
+    /// println!("{}", s.get_physical_core_numbers());
+    /// ```
+    fn get_physical_core_numbers(&self) -> u64;
```

Comment:
What do you think of `get_physical_core_count` instead?

Reply:
I started with `usize` but because all the other function returned `u64` changed it also to `u64`  
Will change